src/Sylius/Bundle/ApiBundle/Serializer/HydraErrorNormalizer.php

```diff
@@ -35,13 +36,16 @@ public function normalize($object, $format = null, array $context = [])
     public function supportsNormalization($data, $format = null)
     {
         if (method_exists($this->requestStack, 'getMainRequest')) {
-            $path = $this->requestStack->getMainRequest()->getPathInfo();
+            $request = $this->requestStack->getMainRequest();
         } else {
-            /** @phpstan-ignore-next-line */
-            $path = $this->requestStack->getMasterRequest()->getPathInfo();
+            $request = $this->requestStack->getMasterRequest();
         }
 
-        if (!str_starts_with($path, $this->newApiRoute)) {
+        if (null === $request) {
+            return false;
```

Comment:
Shouldn't we assert here, rather than continue as missing master request is an alright situation?